src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: 'Avenir Book', sans-serif;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  display: block;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  box-sizing: border-box;
+}
+
+.search-bar__big {
+  height: 70px;
```

Comment:
You have fixed the height of the 'search-bar__big' class. Unless there is a requirement, it is recommended not to fix the container size. Let the content size dictate it to avoid overflow or accidental scroll bars.